src/rest.rs

```diff
@@ -651,7 +641,7 @@ fn handle_request(
     info!("handle {:?} {:?}", method, uri);
     match (
         &method,
-        path.get(0),
+        path.first(),
```

Comment:
lol

I don't mind this, but some people might prefer get(0) for looks (to match the following lines.)